install-rhel-deps-build-openjdk11.sh

```diff
@@ -55,6 +55,7 @@ binutils
 cups-devel
 fontconfig
 freetype-devel
+devtoolset-8
 giflib-devel
 gcc-c++
```

Comment:
Should this be `devtoolset-8-gcc-c++`?

Reply:
I think for later step we use scl enable devtoolset-8 -- configure and  scl enable devtoolset-8 -- make , so we have to install devtoolset-8  think

Reply:
I mean instead of `gcc-c++` install `devtoolset-8-gcc-c++` :)

Reply:
yes that is right :) I will do that